velox/docs/develop/types.rst

```diff
@@ -0,0 +1,55 @@
+=====
+Types
+=====
+
+Velox supports scalar types and complex types. These types cover both
```

Comment:
Would it make sense to explain the difference between scalar and complex types? E.g. complex types contain other types.

Reply:
Complex Types definitely need more love. I plan to focus on Complex Types with examples in a follow-up PR.
Is that okay?